mono/metadata/metadata-internals.h

```diff
@@ -258,16 +260,16 @@ struct _MonoImage {
 	MonoAssembly **references;
 	int nreferences;
 
-	/* Code files in the assembly. */
+	/* Code files in the assembly. The main assembly has a "file" table and also a "module"
+	 * table, where the module table is a subset of the file table. We track both lists,
+	 * and because we can lazy-load them at different times we reference-increment both.
+	 */
 	MonoImage **modules;
 	guint32 module_count;
 	gboolean *modules_loaded;
```

Comment:
That makes sense. Is there a reason we do this only for the "modules" array and not for the "files" array though?